app/controllers/app_controller.rb

```diff
@@ -13,8 +13,14 @@ class TodoApp < Sinatra::Base
     erb :'home/index', :layout => :'layouts/layout'
   end
 
+  # get '/all-tasks/:id/:name' do 
```

Comment:
Try to remember to remove commented out code before opening a pr!

Reply:
😅 👍